docs/jsonnet/expected-structure.md

```diff
@@ -0,0 +1,100 @@
+# Expected structure
+
+Tanka evaluates the `main.jsonnet` file of your [Environment](/environments) and
+filters the output (either Object or Array) for valid Kubernetes objects.  
+An object is considered valid if it has both, a `kind` and a `metadata.name` set.
+
+## Deeply nested object (Recommended)
+The most commonly used structure is a single big object that includes all of
+your configs to be applied to the cluster nested under keys.  
+How deeply encapsulated the actual object is does not matter, Tanka will
+traverse down until it finds something that has both, a `kind` and a
+`metadata.name`.  
+
+??? Example
```

Comment:
Should this be here?

Reply:
Yeah, special markup from `mkdocs`, our docs generator